relay-monitors/src/lib.rs

```diff
@@ -100,6 +100,17 @@ struct MonitorConfig {
     timezone: Option<String>,
 }
 
+// XXX(epurkhiser): This is a duplicate of the ClientSdkInfo that is part of the relay-general
+// crate. Until we're able to migrate the checkin payload over to it's own protocol using
+// metastructure we'll need to have this duplicated here
+//
+/// The SDK Interface describes the Sentry SDK and its configuration used to capture and transmit an event.
+#[derive(Debug, Deserialize, Serialize)]
+struct MinimalClientSdkInfo {
+    pub name: String,
+    pub version: String,
```

Comment:
Are we sure these two values will always be there? If not, I would make them optional: 

```suggestion
    #[serde(default, skip_serializing_if = "Option::is_none")]
    pub name: Option<String>,
    #[serde(default, skip_serializing_if = "Option::is_none")]
    pub version: Option<String>,
```

Reply:
Both of these fields are required in the `ClientSdkInfo` struct

https://github.com/getsentry/relay/blob/3ffdaf1c108ab44d1fea6f73874ddee351975e5d/relay-general/src/protocol/clientsdk.rs#L19-L38